docs/plugins/stockpiles.rst

```diff
@@ -6,7 +6,8 @@ stockpiles
     :tags: fort design productivity stockpiles
 
 Commands act upon the stockpile selected in the UI unless another stockpile
-identifier is specified on the commandline.
+identifier is specified on the commandline. See `gui/stockpiles` for an
```

Comment:
`gui/stockpiles` isn't ready for primetime yet. it needs a full rewrite for the new `stockpiles` functionality